ibis/pandas/udf.py

```diff
@@ -529,10 +528,8 @@ def wrapped(*args):
 
 
 @pre_execute.register(ops.ElementWiseVectorizedUDF)
-@pre_execute.register(ops.ElementWiseVectorizedUDF, PandasClient)
-def pre_execute_elementwise_udf(
-    op, *clients, scope=None, aggcontet=None, **kwargs
-):
+@pre_execute.register(ops.ElementWiseVectorizedUDF, ibis.client.Client)
```

Comment:
doesn't this catch more generically here? I guess we don't have ElementWiseVector UDFs on other clients atm, but why this change?

Reply:
This isn't really necessary but follows the same convention as https://github.com/ibis-project/ibis/blob/master/ibis/pandas/dispatch.py#L49